### DISCUSSION TITLE
Considering a slight Drowned swim nerf

### DISCUSSION THREAD

**Discussion by SirSquidly** [SirSquidly/Oceanic-Expanse#43] [Announcements]
Drowned are very fast when it comes to reaching targets in the water, so I am considering slowing down their vertical swim speed.

Feedback will be appreciated on if this is desired or not.

Please use the Thumbs UP or DOWN to give your input, or leave a comment.

**Comment by Sethgno** [comment]
compared to an unenhanced swimming speed player they effectively feel too fast to me too
idk how changing the ai works but i can imagine them being better if the "response time" of updating the vertical movement were nerfed a little so they arent always on your level, but the actual ascension speed could remain more or less the same so they can still catch up.
or maybe give them slightly randomized intervals of upward movement speed. but idk how they supposed to work im just vibin
